examples/foomedical/src/pages/account/Profile.tsx

```diff
@@ -36,18 +82,33 @@ export function Profile(): JSX.Element | null {
                   data={['', 'female', 'male', 'other', 'unknown']}
                 />
                 <TextInput label="Birth Date" name="birthDate" type="date" defaultValue={profile.birthDate} />
+                <Button type="submit" mr="auto">
+                  Save
+                </Button>
               </Stack>
             </Box>
           </InfoSection>
           <InfoSection title="Contact Information">
-            <Box p="xl" w={500}>
+            <Box p="xl">
               <Stack>
                 <TextInput
                   label="Email"
                   name="email"
                   defaultValue={profile.telecom?.find((t) => t.system === 'email')?.value}
+                  disabled
```

Comment:
I disabled the email field because I wanted to confirm whether or not we will allow people to edit it in the example. Please let me know if I should change this.

Reply:
This is a good idea. Leave as disabled